### DISCUSSION TITLE
Toggle Collapse?

### DISCUSSION THREAD

**Discussion by NaityDhim** [saadeghi/daisyui#1250] [Q&A] [answered]
Hi,

I would like the Collapse to be abble to toggle (open or close by clicking on the + or the v) and not reclose everytime I click somewhere. Is this possible?

**Comment by saadeghi** [comment, marked as answer]
Yes. There's an example in the docs: [Collapse with checkbox](https://daisyui.com/components/collapse/#collapse-with-checkbox)

**Reply by NaityDhim** [reply]
Thanks, this should help :)
